source/reference/object.rst

```diff
@@ -272,7 +272,7 @@ language.
       "001 invalid": "value"
     }
 
-Since object keys must always be strings anyway, so it is implied that the
+Since object keys must always be strings, it is implied that the
```

Comment:
I think the "anyway" is important, but, yes, the "so" is an error.

```suggestion
Since object keys must always be strings anyway, it is implied that the
```